api/krusty/fnplugin_test.go

```diff
@@ -629,3 +630,144 @@ metadata:
   name: env
 `)
 }
+
+func TestFnContainerMounts(t *testing.T) {
+	skipIfNoDocker(t)
+
+	th := kusttest_test.MakeHarness(t)
+	o := th.MakeOptionsPluginsEnabled()
+	fSys := filesys.MakeFsOnDisk()
+	b := krusty.MakeKustomizer(&o)
+	tmpDir, err := filesys.NewTmpConfirmedDir()
+	assert.NoError(t, err)
+
+	path, err := os.Getwd()
+	assert.NoError(t, err)
+	chartPath := filepath.Join(path, "testdata", "charts")
+
+	assert.NoError(t, fSys.WriteFile(filepath.Join(tmpDir.String(), "kustomization.yaml"), []byte(`
+generators:
+  - |-
+    apiVersion: v1alpha1
+    kind: RenderHelmChart
+    metadata:
+      name: demo
+      annotations:
+        config.kubernetes.io/function: |
+          container:
+            image: gcr.io/kpt-fn/render-helm-chart:v0.1.0
```

Comment:
This makes our CI depend on this Kpt container continuing to be published here, right? Can you open an issue about switching to the donated one once it is available?

Reply:
> This makes our CI depend on this Kpt container continuing to be published here, right? Can you open an issue about switching to the donated one once it is available?

This was on my mind too, I can file an issue to make sure we don't forget. Though this is not a problem until we start to run docker tests in our CI. I'll file an issue for that as well.